controllers/RegistrationController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace app\controllers;
+
+use Yii;
+use app\models\forms\RegistrationForm;
+use yii\filters\AccessControl;
+
+class RegistrationController extends \yii\web\Controller
+{
+    public function behaviors()
+    {
+        return [
+            'access' => [
+                'class' => AccessControl::class,
+                'only' => ['index'],
+                'rules' => [
+                    [
+                        'allow' => true,
+                        'actions' => ['index'],
+                        'roles' => ['?']
+                    ]
+                ]
+            ]
+        ];
+    }
+
+    public function actionIndex()
+    {
+        $registrationForm = new RegistrationForm();
+
+        if (Yii::$app->request->getIsPost()) {
+            $registrationForm->load(Yii::$app->request->post());
+        }
+
+        if ($registrationForm->validate()) {
+
+            $registrationForm->createUser();
+
+            return $this->goHome();
+        }
```

Comment:
Из метода `createUser` лучше возвращаться `bool`. Во время создания пользователя могут возникнуть ошибки (к примеру, валидация не пройдёт), модель пользователя создана не будет, но тем не менее код в случае такой ошибки продолжит своё выполнение по пути успешного сценария

```suggestion
        if ($registrationForm->validate() && $registrationForm->createUser()) {
            return $this->goHome();
        }
```